src/language-js/parser-babel.js

```diff
@@ -129,6 +129,20 @@ function tryCombinations(fn, combinations) {
   throw error;
 }
 
+function rethrowSomeRecoveredErrors(ast) {
+  if (ast.errors) {
+    for (const error of ast.errors) {
+      if (
+        typeof error.message === "string" &&
+        error.message.startsWith("Did not expect a type annotation here.")
```

Comment:
```suggestion
        error.message === "Did not expect a type annotation here."
```

We have tests, let's use simple test?

Reply:
The full message also includes `(${loc.line}:${loc.column})`.

Reply:
Okay.